### PR TITLE
config: allow for artifacts being mentioned in comment

### PR DESCRIPTION
**Pull request by miri64**
As discussed in RIOT-CI Matrix chat :-).

### CODE REVIEW

**Comment by aabadie** [comment]
So if I understand correctly, you want to have a configurable list of artifacts displayed in the comment. Why not just take it from the job ? They are also configured in the .murdock.yml file. That would make the thing simpler.

File: murdock/templates/comment.md.j2 [hunk 1]
```diff
@@ -38,6 +38,15 @@
 
 </details>
 {% endif %}
+{% if job.config.pr.comment_artifacts %}
+
+#### Artifacts
+{% for artifact in job.config.pr.comment_artifacts %}
+{% if artifact.name in job.artifacts %}
+- [{{ artifact.readable_name }}]({{ job.http_dir }}/{{ artifact.name }})
```
Comment (aabadie) [inline murdock/templates/comment.md.j2:46]: The `job.http_dir` is not enough, it doesn't contain the base url. You probably have to extends the context of the template with `GLOBAL_CONFIG.base_url`:

https://github.com/murdock-ng/murdock/blob/main/murdock/github.py#L73

woud become:

```python
context = {"job": job, "base_url": GLOBAL_CONFIG.base_url}
```

And then this line:
```suggestion
- [{{ artifact.readable_name }}]({{ base_url }}/{{ job.http_dir }}/{{ artifact.name }})
```

Comment (miri64) [reply murdock/templates/comment.md.j2:46]: Mhhh I was copying this from somewhere where the URL is used as well. Maybe I just need to fix the template ;-).

**Comment by miri64** [comment]
> Why not just take it from the job ? They are also configured in the .murdock.yml file. That would make the thing simpler.

Because I want to have be able to filter the artifacts that are linked (e.g. adding `output.txt` seems redundant to me) and I want (optional) readable names for the artifacts.

File: murdock/templates/comment.md.j2 (hunk 1 above)
Comment (miri64) [reply murdock/templates/comment.md.j2:46]: Ah no, you are right!

**Comment by aabadie** [comment]
> Because I want to have be able to filter the artifacts that are linked (e.g. adding output.txt seems redundant to me) and I want (optional) readable names for the artifacts.

Makes sense!

**Comment by miri64** [comment]
Still working on tests, btw.

**Comment by miri64** [comment]
Tests added.

**Review by aabadie — APPROVED** [review]

**Comment by miri64** [comment]
Thanks for the review! :-)

**Pull request merged**